mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -394,7 +394,7 @@ async fn request_and_fill_orderbook(ctx: &MmArc, base: &str, rel: &str) -> Resul
 
     let ordermatch_ctx = OrdermatchContext::from_ctx(ctx).unwrap();
     let mut orderbook = ordermatch_ctx.orderbook.lock();
-
+    let my_zhtlc_orders_pubkeys = orderbook.my_zhtlc_orders_pubkeys.clone();
```

Comment:
I did it because orderbook [is used further by mutably reference](https://github.com/KomodoPlatform/atomicDEX-API/pull/1849/files/7742d52e4c30a3a67f93c9c05af47790c853d22c#diff-4c4940ed0030a5a26cc593870df06f290339e4ed69d0e7564b2c71135e4d2787R428) and decided to not start more big refactoring here in this PR.

Reply:
Still don't get it, it was working fine before without cloning and by passing a reference to `orderbook.my_zhtlc_orders_pubkeys` and the orderbook mutex stays locked anyways throughout the function. Cloning is just using more memory in this case and nothing more.